pkgs/by-name/pr/pre-commit/package.nix

```diff
@@ -17,6 +17,9 @@
 }:
 
 with python3Packages;
+let
+  i686Linux = python3Packages.stdenv.buildPlatform.system == "i686-linux";
```

Comment:
Use stdenv from pkgs aka no prefix

Reply:
I removed the `python3Packages.` prefix. This is still the same thing as `python3Packages.stdenv` because of the `with python3Packages;` which I didn’t notice before. Is that what you wanted, or should I add `stdenv` to the list of function parameters at the top of the file?